handlers/main.yml

```diff
@@ -18,6 +18,17 @@
     - openhpc_slurmdbd_host in play_hosts
   run_once: true
```

Comment:
IIUC, I think there already is one here: https://github.com/stackhpc/ansible-role-openhpc/blob/e9dac90810cda181d87ac7c94bc25980344787d6/handlers/main.yml#L10-L19, just with a different name!

Reply:
That's what I'm saying: the restart one at line 10 should have a `listen: Restart slurmdbd service` added and the name changed to `name: Issue slurmdbd restart command`. Then when you do `notify: Restart slurmdbd service` its more obvious you're getting two listen-trigged tasks, rather than having to realise that gets both the restart task (by name) and the wait task (by listen). That make sense?

Reply:
Yup, you're right and I learnt something new about Ansible handlers.